src/MessageText.js

```diff
@@ -21,14 +16,19 @@ export default class MessageText extends React.Component {
     this.onEmailPress = this.onEmailPress.bind(this);
   }
 
+  /**
+   * When someone sends a message that includes a website address beginning with "www." (omitting the scheme),
+   * react-native-parsed-text recognizes it as a valid url, but Linking fails to open due to the missing scheme.
+   * @param {string} url
+   * @memberof MessageText
+   */
   onUrlPress(url) {
-    // When someone sends a message that includes a website address beginning with "www." (omitting the scheme),
-    // react-native-parsed-text recognizes it as a valid url, but Linking fails to open due to the missing scheme.
```

Comment:
I think this comment should be left where it was; it's describing specific logic rather than the method itself.

Reply:
Fixed in commit ce91bf33b14b0894d5daca1ba47b46199889e7b4